examples/postgresql/assembly/index.ts

```diff
@@ -0,0 +1,34 @@
+import { postgresql } from "@hypermode/functions-as";
+
+export function queryPeople(): string {
+  return postgresql.query("neon", "select * from people", []);
+}
+
+export function getPerson(name: string): string {
+  const query = "select * from people where name = $1";
+  return postgresql.query("neon", query, [name]);
+}
+
+export function addPerson(
+  name: string,
+  age: i32,
+  balance: f64,
+  privkey: i8[],
```

Comment:
```suggestion
  privkey: u8[],
```

Or:

```suggestion
  privkey: Uint8Array,
```

Or:
```suggestion
  privkey: ArrayBuffer,
```

Ideally the runtime should support all three (we should test that), but either way it would be unsigned.